frontend/views/components/GroupWelcome.vue

```diff
@@ -52,6 +54,14 @@ export default ({
     toDashboard () {
       if (this.isButtonClicked) return
       this.isButtonClicked = true
+      const groupContractID = this.currentGroupId
```

Comment:
I'm wondering whether this should be here, or if instead it should be done on the `/dashboard` page on `onMounted`. Maybe it doesn't matter.

Reply:
Pros of doing it here: it's only done once (for users). Pros of doing it in the `/dashboard` page: we don't need to change the `commands.js` in the tests. Cons of doing it in the `/dashboard` page: need to do this check every time.